solidity/contracts/isms/wormhole/InterchainAccountWormholeIsm.sol

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+pragma solidity >=0.8.0;
+
+// ============ Internal Imports ============
+import {AbstractWormholeIsm} from "./AbstractWormholeIsm.sol";
+import {Message} from "../../libs/Message.sol";
+import {InterchainAccountMessage} from "../../libs/middleware/InterchainAccountMessage.sol";
+
+contract InterchainAccountWormholeIsm is AbstractWormholeIsm {
```

Comment:
I think the interchain account stuff should definitely be a separate PR

Reply:
This was @asaj's original PR I was using as reference, I can roll it back.